source/c_datamanagement.rst

```diff
@@ -0,0 +1,779 @@
+	stop_level_delivery_side,	double precision
+	image_url,	character varying
+	geometry,	USER-DEFINED
+	end,	timestamp with time zone
+	start,	timestamp with time zone
+	num_timeseries,	integer
+	last_modified,	timestamp with time zone
+    
+.. csv-table:: "regions": "https://demo.lizard.net/api/v3/regions/",
+    :header: column_name, data_type
+
+    id, integer
+    
+.. csv-table:: "roads": "https://demo.lizard.net/api/v3/roads/",
+    :header: column_name, data_type
+    
+	id,	integer
+	created,	timestamp with time zone
+	name,	character varying
+	type,	integer
+	use,	integer
+	geometry,	USER-DEFINED
+	code,	character varying
+	end,	timestamp with time zone
+	image_url,	character varying
+	organisation_id,	integer
+	start,	timestamp with time zone
+	region_id,	integer
+	last_modified,	timestamp with time zone
+
+.. csv-table:: "sluices": "https://demo.lizard.net/api/v3/sluices/",
+    :header: column_name, data_type
+
+	id,	integer
+	organisation_id,	integer
+	created,	timestamp with time zone
+	code,	character varying
+	name,	character varying
+	image_url,	character varying
+	geometry,	USER-DEFINED
+	end,	timestamp with time zone
+	start,	timestamp with time zone
+	num_timeseries,	integer
+	last_modified,	timestamp with time zone
+    
+.. csv-table:: "wastewatertreatmentplants": "https://demo.lizard.net/api/v3/wastewatertreatmentplants/",
+    :header: column_name, data_type
+
+	id,	integer
+	organisation_id,	integer
+	created,	timestamp with time zone
+	code,	character varying
+	name,	character varying
+	image_url,	character varying
+	geometry,	USER-DEFINED
+	end,	timestamp with time zone
+	start,	timestamp with time zone
+	num_timeseries,	integer
+	last_modified,	timestamp with time zone
+
+.. csv-table:: "weirs": "https://demo.lizard.net/api/v3/weirs/",
+    :header: column_name, data_type
+
+    id	integer
+	organisation_id,	integer
+	created,	timestamp with time zone
+	code,	character varying
+	crest_type,	smallint
+	crest_width,	double precision
+	crest_level,	double precision
+	name,	character varying
+	lat_dis_coeff,	double precision
+	angle,	double precision
+	allowed_flow_direction,	integer
+	controlled,	integer
+	comment,	text
+	discharge_coeff,	double precision
+	image_url,	character varying
+	geometry,	USER-DEFINED
+	end,	timestamp with time zone
+	start,	timestamp with time zone
+	num_timeseries,	integer
+	type,	smallint
+	last_modified,	timestamp with time zone
+
+Rasters
+-------
+
+Rasters in Lizard represent continuous information about the physical environment across an area.
+Rasters can be static or temporal. Examples of static data are digital elevation models and land cover maps.
+Temporal raster datasets, or raster series, consist of a series of rasters for a certain time interval.
+Examples of temporal raster datasets are radar measurements of precipitation, air quality or hydrodynamic model results such as flood depths. 
+
+Our Raster Store enables you to store and accelerate your raster data sets.
```

Comment:
"accellerate" anders in de zin zetten. "store and accelerate", dan gooi je je raster data met hoge snelheid het raam uit.